maintaining.md

```diff
@@ -0,0 +1,7 @@
+# Maintaining
+
+## Developing Kap
+
+Run `npm run dev` in one terminal tab to start watch mode, and in another tab, run `npm start` to launch Kap.
+
+We strongly recommend installing an [XO editor plugin](https://github.com/sindresorhus/xo#editor-plugins) for JavaScript linting and [Stylelint editor plugin](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/complementary-tools.md#editor-plugins) for CSS linting. Both of these support auto-fix on save.
```

Comment:
"and **a** [Stylelint editor plugin](..." maybe?

Reply:
Sure